connector_ecommerce/models/product.py

```diff
@@ -29,19 +12,23 @@ class ProductTemplate(models.Model):
 
     # TODO implement set function and also support multi tax
     @api.one
-    @api.depends('taxes_id', 'taxes_id.group_id')
-    def _get_tax_group_id(self):
+    @api.depends('taxes_id.tax_group_id')
+    def _compute_tax_group_id(self):
         taxes = self.taxes_id
-        self.tax_group_id = taxes[0].group_id.id if taxes else False
+        self.tax_group_id = taxes[0].tax_group_id.id if taxes else False
 
     tax_group_id = fields.Many2one(
         comodel_name='account.tax.group',
```

Comment:
Hi, 
Where is the definition of this model ? Don't you forgot to git add some file ?